### PR TITLE
Add more graphs & refine old ones

### PR DESCRIPTION
**Pull request by songofroland**

### CODE REVIEW

File: src/data/utils.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {Graph} from '../logic/commonInterfaces';
+
+export function adjMatrixToGraphString(matrix: Graph){
+  return matrix.map(row => row.join(' ')).join('\n');
```
Comment (wilmhit) [inline src/data/utils.ts:4]: Single spaces are not really cutting it. We should use single spaces if there are only single-digit numbers in matrix. When there is even one number >9, function should start aligning graph with double spaces. Graph parsing handles that well.

File: src/data/graphSamples.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    graph: [
+      [0, 1, 2, 3, 4, 5, 6],
+      [1, 0, 0, 0, 0, 0, 0],
+      [2, 0, 0, 0, 0, 0, 0],
+      [3, 0, 0, 0, 0, 0, 0],
+      [4, 0, 0, 0, 0, 0, 0],
+      [5, 0, 0, 0, 0, 0, 0],
+      [6, 0, 0, 0, 0, 0, 0],
+    ],
+    string: '0 1 2 3 4 5 6\n1 0 0 0 0 0 0\n2 0 0 0 0 0 0\n3 0 0 0 0 0 0\n4 0 0 0 0 0 0\n5 0 0 0 0 0 0\n6 0 0 0 0 0 0',
```
Comment (wilmhit) [inline src/data/graphSamples.ts:50]: Now let's use `adjMatrixToGraphString` here

File: src/data/graphSamples.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+/* eslint-disable */
```
Comment (wilmhit) [inline src/data/graphSamples.ts:1]: Delete this when there is no more `line too long` warnings.

File: src/components/GraphEditor.tsx [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
     key: 'simple',
   },
   {
-    label: 'Medium graph',
+    label: 'Medium',
     key: 'medium',
   },
   {
-    label: 'Advanced graph',
+    label: 'Neuron',
```
Comment (wilmhit) [inline src/components/GraphEditor.tsx:20]: Intresting name, doesn't tell much, but I didn't see the graph as of now.

File: src/test/utils.test.ts [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import {adjMatrixToGraphString as toGraphString} from '../data/utils';
+import samples from '../data/graphSamples';
+
+test('Makes valid graph string' ,() => {
+  expect(toGraphString(samples.advanced.graph)).toStrictEqual(samples.advanced.string);
+  expect(toGraphString(samples.dense.graph)).toStrictEqual(samples.dense.string);
+  expect(toGraphString(samples.neuron.graph)).toStrictEqual(samples.neuron.string);
+});
```
Comment (wilmhit) [inline src/test/utils.test.ts:8]: \n

File: src/data/utils.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import {Graph} from '../logic/commonInterfaces';
```
Comment (wilmhit) [inline src/data/utils.ts:1]: I think this file would fit more in `logic` folder.

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/data/utils.ts (hunk 6 above)
Comment (songofroland) [reply src/data/utils.ts:1]: I'd leave it here since it is used  only with the samples

File: src/data/graphSamples.ts (hunk 2 above)
Comment (songofroland) [reply src/data/graphSamples.ts:50]: wtf why we did not do that from the beginning

Comment (wilmhit) [reply src/data/graphSamples.ts]: `adjMatrixToGraphString` did not exist

File: src/data/utils.ts (hunk 1 above)
Comment (wilmhit) [reply src/data/utils.ts:7]: That still isn't algorithm that would align it correctly. Now that i looked at the samples we don't need it.

**Review by ghost — APPROVED** [review]

**Pull request merged**